cell_classification/plot_utils.py

```diff
@@ -188,6 +190,105 @@ def plot_metrics_against_threshold(
     plt.close()
 
 
+def collapse_activity_dfs(pred_list):
+    """Collapse activity dataframes
+    Args:
+        pred_list (list):
+            list of dictionaries containing activity dataframes
+    Returns:
+        collapsed_df (pd.DataFrame):
+            collapsed activity dataframe
+    """
+    collapsed_df = pd.DataFrame()
+    for sample in pred_list:
+        collapsed_df = pd.concat([collapsed_df, sample["activity_df"]])
```

Comment:
It's faster to make a list and then perform a single concatenate call at the end https://github.com/angelolab/ark-analysis/issues/751